sdks/python/apache_beam/pvalue.py

```diff
@@ -82,7 +82,7 @@ class PValue(object):
   """
 
   def __init__(self,
-               pipeline,  # type: Pipeline
+               pipeline,  # type: Optional[Pipeline]
```

Comment:
It's a bit unclear to me.  Would love some more insight from Beam experts.

Reply:
The one case where it could be None is when we're deserializing these from a runner api proto (which you reference below). This is because there is a cycle between PTransforms and their outputs, and the pipeline as a whole and its components. It should be populated later and as such documenting as optional isn't really correct. Is it possible to instead suppress the warning at the callsite?

Reply:
> Is it possible to instead suppress the warning at the callsite?

yes, if the state where the pipeline is `None` is short-lived then this is probably preferable to adding lots more assertions.

Reply:
Let's do that then.

Reply:
done